src/backend/TrafficCourts/Citizen.Service/Features/Tickets/Analyse.cs

```diff
@@ -0,0 +1,117 @@
+using Azure;
+using Azure.AI.FormRecognizer.DocumentAnalysis;
+using MediatR;
+using Microsoft.Extensions.Options;
+using TrafficCourts.Citizen.Service.Configuration;
+
+namespace TrafficCourts.Citizen.Service.Features.Tickets
+{
+    public static class Analyse
+    {
+        public class AnalyseRequest : IRequest<AnalyseResponse>
+        {
+            public AnalyseRequest(IFormFile image)
+            {
+                this.image = image;
+            }
+
+            public IFormFile image { get; set; }
+        }
+
+        public class AnalyseResponse
+        {
+            public AnalyseResponse(ViolationTicket violationTicket)
+            {
+                this.violationTicket = violationTicket;
+            }
+
+            public ViolationTicket? violationTicket { get; set; }
+        }
+
+        public class Handler : IRequestHandler<AnalyseRequest, AnalyseResponse>
+        {
+            private readonly ILogger<Handler> _logger;
+            private readonly String _apiKey;
+            private readonly String _endpoint;
+
+            public Handler(ILogger<Handler> logger, IOptions<FormRecognizerConfigurationOptions> options)
+            {
+                _logger = logger ?? throw new ArgumentNullException(nameof(logger));
+                _apiKey = String.IsNullOrEmpty(options.Value.ApiKey) ? throw new ArgumentNullException(nameof(options.Value.ApiKey)) : options.Value.ApiKey;
+                _endpoint = String.IsNullOrEmpty(options.Value.Endpoint) ? throw new ArgumentNullException(nameof(options.Value.Endpoint)) : options.Value.Endpoint;
+            }
+
+            public async Task<AnalyseResponse> Handle(AnalyseRequest request, CancellationToken cancellationToken)
+            {
+                _logger.LogDebug($"Analysing {request.image.FileName}");
+
+                AzureKeyCredential credential = new AzureKeyCredential(_apiKey);
+                DocumentAnalysisClient client = new DocumentAnalysisClient(new Uri(_endpoint), credential);

```

Comment:
The configuration value could be a Uri. If the _endpoint is not well formed as a uri, this will throw FormatException. the value comes from configuration.